integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -2084,9 +2088,7 @@ msgstr "wird derzeit übersetzt"
 #: cms/models/abstract_content_translation.py:48
 msgid ""
 "Flag to indicate a translation is being updated by an external translator"
-msgstr ""
-"Feld, die anzeigt, dass eine Übersetzung von einem externen Übersetzer "
-"aktualisiert wird"
+msgstr "Flagge, die anzeigt, dass eine Übersetzung extern aktualisiert wird"
```

Comment:
```suggestion
msgstr "Feld, das anzeigt, dass eine Übersetzung extern aktualisiert wird"
```
_Flagge_ in Gerrman is not really a term for a boolean field, right? If you want a more literal translation, I'd rather change the english text to "Field [...]".